backend/migrations/versions/799a2343955e_fix_moduletype_enum.py

```
@@ -0,0 +1,118 @@
+        batch_op.alter_column(
+            "module_type_temp",
+            new_column_name="module_type",
+            existing_type=new_module_type,
+            nullable=False,
+        )
+
+    # Drop the old enum type
+    old_module_type.drop(op.get_bind(), checkfirst=False)
+
+
+def downgrade():
```

Ruff (0.8.2)

Missing return type annotation for public function `downgrade`

Add return type annotation: `None`

(ANN201)